### DISCUSSION TITLE
Hide App Title & Traffic lights

### DISCUSSION THREAD

**Discussion by niravbhimani53** [hecrj/iced#1303] [Q&A] [answered]
Hello fellow devs,

Can we hide the traffic light (i.e, Close, Minimize, Fullscreen icon) & App title text?

Cheers

<img width="1050" alt="Screen Shot 2022-04-11 at 10 53 00 pm" src="https://user-images.githubusercontent.com/12537912/162743831-1e94c3f8-4bdd-4f3c-a009-3a62c2d61a6b.png">

**Comment by hecrj** [comment, marked as answer]
You can try setting the [`window::Settings::decorations`](https://docs.rs/iced/latest/iced/window/struct.Settings.html#structfield.decorations) to `false`.

**Reply by niravbhimani53** [reply]
Cheers. I was searching for traffic lights in the document. No wonder I didn't get any results.
